.github/workflows/ci.yml

```
@@ -18,6 +19,7 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 22
+          cache: 'npm' # Ajout du cache pour les dépendances npm
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)